tests/pc/test_lag_member_forwarding.py

```diff
@@ -0,0 +1,175 @@
+import ipaddr as ipaddress
+import json
+import pytest
+import time
+from tests.common import config_reload
+from ptf.mask import Mask
+import ptf.packet as scapy
+import ptf.testutils as testutils
+from tests.common.helpers.assertions import pytest_assert
+
+pytestmark = [
+    pytest.mark.topology('any')
+]
+
+
+def build_pkt(dest_mac, ip_addr, ttl):
+    pkt = testutils.simple_tcp_packet(
+          eth_dst=dest_mac,
+          eth_src="00:11:22:33:44:55",
+          pktlen=100,
+          ip_src="19.0.0.100",
+          ip_dst=ip_addr,
+          ip_ttl=ttl,
+          tcp_dport=200,
+          tcp_sport=100
+    )
+    exp_packet = Mask(pkt)
+    exp_packet.set_do_not_care_scapy(scapy.Ether, "dst")
+    exp_packet.set_do_not_care_scapy(scapy.Ether, "src")
+
+    exp_packet.set_do_not_care_scapy(scapy.IP, "version")
+    exp_packet.set_do_not_care_scapy(scapy.IP, "ihl")
+    exp_packet.set_do_not_care_scapy(scapy.IP, "tos")
+    exp_packet.set_do_not_care_scapy(scapy.IP, "len")
+    exp_packet.set_do_not_care_scapy(scapy.IP, "flags")
+    exp_packet.set_do_not_care_scapy(scapy.IP, "id")
+    exp_packet.set_do_not_care_scapy(scapy.IP, "frag")
+    exp_packet.set_do_not_care_scapy(scapy.IP, "ttl")
+    exp_packet.set_do_not_care_scapy(scapy.IP, "chksum")
+    exp_packet.set_do_not_care_scapy(scapy.IP, "options")
+
+    exp_packet.set_do_not_care_scapy(scapy.TCP, "seq")
+    exp_packet.set_do_not_care_scapy(scapy.TCP, "ack")
+    exp_packet.set_do_not_care_scapy(scapy.TCP, "reserved")
+    exp_packet.set_do_not_care_scapy(scapy.TCP, "dataofs")
+    exp_packet.set_do_not_care_scapy(scapy.TCP, "window")
+    exp_packet.set_do_not_care_scapy(scapy.TCP, "chksum")
+    exp_packet.set_do_not_care_scapy(scapy.TCP, "urgptr")
+
+    exp_packet.set_ignore_extra_bytes()
+    return pkt, exp_packet
+
+
+def test_lag_member_forwarding_packets(duthosts, enum_rand_one_per_hwsku_frontend_hostname, tbinfo, ptfadapter):
+    duthost = duthosts[enum_rand_one_per_hwsku_frontend_hostname]
+    mg_facts = duthost.get_extended_minigraph_facts(tbinfo)
+    lag_facts = duthost.lag_facts(host=duthost.hostname)['ansible_facts']['lag_facts']
+    if not len(lag_facts['lags'].keys()):
+        pytest.skip("No Lag found in this topology")
+    portchannel_name = list(lag_facts['lags'].keys())[0]
+    portchannel_dest_name = None
+    recv_port = []
+    if len(lag_facts['lags'].keys()) > 1:
```

Comment:
should the check here be for > 0? Also what if this check fails?

Reply:
@prabhataravind : this check is just trying to find 2 port channels in system. If we don't find 2nd portchannel out of 3 cases mention in PR description we will not verify 3rd one ` Data packets across portchannel not to be processed ` . 
Other 2 scenario will still be validated and test should run accordingly.